manifest.yml

```diff
@@ -11,7 +11,7 @@ dependencies:
       path: "FreeRTOS/Source"
 
   - name: "FreeRTOS-Plus-TCP"
-    version: "67b9e1c"
+    version: "40c16fe"
```

Comment:
Is this change merged? I can not see in main branh

Reply:
Yes, [PR#990](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/pull/990) in FreeRTOS-Plus-TCP has been merged.